src/dps-notification-service/src/main/resources/outputNotification.wsdl

```diff
@@ -0,0 +1,88 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<wsdl:definitions name="outputNotificationWS" targetNamespace="http://reeks.bcgov/DPS_Extensions.common.wsProvider:outputNotificationWS" xmlns:soapenc="http://schemas.xmlsoap.org/soap/encoding/" xmlns:http="http://schemas.xmlsoap.org/wsdl/http/" xmlns:soapjms="http://www.w3.org/2010/soapjms/" xmlns:tns="http://reeks.bcgov/DPS_Extensions.common.wsProvider:outputNotificationWS" xmlns:wsdl="http://schemas.xmlsoap.org/wsdl/" xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:mime="http://schemas.xmlsoap.org/wsdl/mime/" xmlns:soap="http://schemas.xmlsoap.org/wsdl/soap/" xmlns:soap12="http://schemas.xmlsoap.org/wsdl/soap12/">
```

Comment:
can we change the target namespace to be the same as the registration service ?

https://github.com/bcgov/jag-dps/blob/43beeb293d8557303aac04e24d93efdc5bfee17e/src/dps-registration-api/src/main/resources/dpsRegistrationService.wsdl#L4

thanks ;-)

Reply:
Done!